### DISCUSSION TITLE
Add option to separate the track map from live timing to its own window.

### DISCUSSION THREAD

**Discussion by Jurtaani** [f1multiviewer/issue-tracker#138] [Feature requests] [answered]
I think that the driver tracker inside live timing app is much clearer than the official broadcast version.
But to use it, we must scroll down and "hide" the live timing info.
I would much rather keep it on my second monitor at full screen, without needing to hide the live timing information.

There is already same option for the race control messages, so why not add similar thing for that map as well.

**Comment by JJWatMyself** [comment]
This would be amazing. Running a second Live Timing window, in essence to have an additional window isn't a good idea, so a native pop-out is needed for sure.

**Comment by LapsTimeOFF** [comment]
Added in 1.6.9 !

**Comment by f1multiviewer** [comment, marked as answer]
Done!
